doc/developers/contributing.rst

```diff
@@ -181,6 +191,16 @@ Contributing code
   so far. In order to claim an issue for yourself, please comment exactly
   ``/take`` on it for the CI to automatically assign the issue to you.
 
+To maintain the quality of the codebase and ease the review process, any
+contribution, including code generated by language models (LLMs), must conform
+to the project's existing style guide and coding conventions, in particular:
+
+- Avoid modifying unrelated lines to keep the diff of the PR focused on the scope
+  stated in its description or linked issue.
```

Comment:
A suggestion to express the same thing less wordy:
```suggestion
To maintain the quality of the codebase and ease the review process, any
contribution must conform to the project's coding conventions, in particular:

- Don't modify unrelated lines to keep the PR focused on the scope stated in its 
description or issue.
```

I believe there is no "project's style guide" this is why I deleted this phrase (if there is, we should of cause keep it and add a link).